wallet/src/de/schildbach/wallet/Extensions.kt

```diff
@@ -1,46 +0,0 @@
-package de.schildbach.wallet
-
-import android.content.Context
-import android.content.ContextWrapper
-import androidx.fragment.app.FragmentActivity
-import androidx.lifecycle.LifecycleOwner
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.Observer
-
-//
-// https://stackoverflow.com/questions/49075413/easy-way-to-get-current-activity-fragment-lifecycleowner-from-within-view
-//
-
-fun Context.fragmentActivity(): FragmentActivity? {
-    var curContext = this
-    var maxDepth = 20
-    while (--maxDepth > 0 && curContext !is FragmentActivity) {
-        curContext = (curContext as ContextWrapper).baseContext
-    }
-    return if(curContext is FragmentActivity)
-        curContext
-    else
-        null
-}
-
-fun Context.lifecycleOwner(): LifecycleOwner? {
-    var curContext = this
-    var maxDepth = 20
-    while (maxDepth-- > 0 && curContext !is LifecycleOwner) {
-        curContext = (curContext as ContextWrapper).baseContext
-    }
-    return if (curContext is LifecycleOwner) {
-        curContext
-    } else {
-        null
-    }
-}
-
-fun <T> LiveData<T>.observeOnce(lifecycleOwner: LifecycleOwner, observer: Observer<T>) {
```

Comment:
These extensions are probably not needed anymore. `observeOnce` is replaced with `suspend` calls and the rest aren't used.